### PR TITLE
export the segmentId and spanId in toolkit

### PR DESCRIPTION
**Pull request by EvanLjp**
export the segmentId and spanId in toolkit for log protocol

### CODE REVIEW

File: apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/context/ContextManager.java [hunk 1]
```diff
@@ -72,11 +74,23 @@ private static AbstractTracerContext get() {
...
-            return segment.getReadablePrimaryTraceId();
-        }
+        return Objects.nonNull(segment) ? segment.getReadablePrimaryTraceId() : EMPTY_TRACE_CONTEXT_ID;
+    }
+
+    /**
+     * @return the current segment id if needEnhance. Otherwise, "N/A".
+     */
+    public static String getSegmentId() {
+        AbstractTracerContext segment = CONTEXT.get();
```
Comment (wu-sheng) [inline apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/context/ContextManager.java:84]: Please rename variable to `context`, seems the previous version is wrong.

**Comment by wu-sheng** [comment]
Basically this PR is good.

**Review by wu-sheng — DISMISSED** [review]
LGTM. Merge if plugin tests pass.

**Review by wu-sheng — APPROVED** [review]

**Review by kezhenxu94 — APPROVED** [review]

**Review by wu-sheng — CHANGES_REQUESTED** [review]
Test case code style issue, please fix

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**